package-lock.json

```diff
@@ -19,9 +19,9 @@
       }
     },
     "@gov.au/animate": {
-      "version": "1.0.9",
-      "resolved": "https://registry.npmjs.org/@gov.au/animate/-/animate-1.0.9.tgz",
-      "integrity": "sha512-KL+k1jprAMbXxYp0RtCp9ETZ523wf+BIR6F48yUNggrIavxfMvpQjyoBwGhxXnPc5wxSQGZkNCD0NWwlLWrFHA==",
+      "version": "1.0.10",
```

Comment:
Running an `npm install` in order to compile the styles will auto-bump any minor changes to npm packages and store that in the package-lock. The styles.css file I commit as part of this ticket is compiled with whatever versions are in the package-lock. 

Generally (in other projects) these package-lock versions would be committed as they occur, but if you'd rather have these committed separately we could do that too. I think it may just run a chance that if a package version difference causes the compiled files to render differently, it may cause some confusion.

Reply:
@tobybellwood Do you have any opinion on this?

Reply:
Given we'll adopt a merge-to-branch strategy for these A11y fixes - we'll probably end up with another final package-lock update at the end - there is always the risk that the @latest module doesn't affect the target of the PR, but may have an adverse effect somewhere else.  I'd suggest that unless the package-lock contains an update to a package that is critical to the PR being raised, it can probably be omitted?